jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/persistence/QueryDefinitionEntity.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.kie.services.impl.query.persistence;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import org.jbpm.kie.services.impl.query.SqlQueryDefinition;
+import org.jbpm.services.api.query.model.QueryDefinition;
+import org.jbpm.services.api.query.model.QueryDefinition.Target;
+
+@NamedQueries(value={
+        @NamedQuery(name="getQueryDefinitions", query="select qd from QueryDefinitionEntity qd"),
+        @NamedQuery(name="getQueryDefinitionByName", query="select qd from QueryDefinitionEntity qd where name = :name")
+})
+@Entity
+@Table(name="QueryDefinitionStore", uniqueConstraints={@UniqueConstraint(columnNames="Q_NAME")})
+@SequenceGenerator(name="queryDefIdSeq", sequenceName="QUERY_DEF_ID_SEQ", allocationSize=1)
+public class QueryDefinitionEntity {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator="queryDefIdSeq")
+    @Column(name = "id")
+    private Long id;
+ 
+    @Lob
+    @Column(name="Q_EXPRESSION", length=65535)
```

Comment:
@mswiderski If possible, it would be better to keep the same naming conventions that we've used for all of our other tables: in other words, camel case starting with a lowercase letter. 

I can understand wanting to make the names different because they might be special, but IMHO, this just leads to more effort in the long run.